project-management-application-app/src/test/java/com/newklearz/controllers/Utils.java

```diff
@@ -0,0 +1,33 @@
+package com.newklearz.controllers;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class Utils
+{
+    public static String getAlphaNumericString()
+    {
+        String generatedString = RandomStringUtils.randomAlphabetic(8);
+        return generatedString;
+    }
+
+    public static String getRandomEmail()
+    {
+        String allowedChars = "abcdefghijklmnopqrstuvwxyz" + "1234567890" + "_-.";
```

Comment:
This can be constant

Reply:
Do i have to get rid of the entire static method and hard-code some values into class member constants or do i have to declare a class member constant and set it with these methods that generate random values ?